domain/model/pokemon.go

```diff
@@ -1,10 +1,7 @@
 package model
 
 // Pokemon model
-// TODO: Can I use csv notations in model?
 type Pokemon struct {
-	ID    uint16 `csv:"id"`
-	Name  string `csv:"name"`
-	Type1 string `csv:"type1"`
-	Type2 string `csv:"type2"`
+	Name string `csv:"name"`
+	Url  string `csv:"url"`
```

Comment:
By convention, the acronyms must be in uppercase -> `URL`

Reply:
Done